mpt_api_client/rql/query_builder.py

```
@@ -9,7 +9,13 @@
 QueryValue = str | bool | dt.date | dt.datetime | Numeric
 
 
-def parse_kwargs(query_dict: dict[str, QueryValue]) -> list[str]:  # noqa: WPS231
+def quote_rql_value(value: str) -> str:
+    """Wrap value in single quotes for RQL comparison operators."""
+    escaped_value = value.replace("'", r"\'")
+    return f"'{escaped_value}'"
+
+
+def parse_kwargs(query_dict: dict[str, QueryValue]) -> list[str]:  # noqa: WPS231 C901
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `C901`; unknown: `WPS231`)

Remove unused `noqa` directive

(RUF100)